demos/mqtt/mqtt_demo_mutual_auth/CMakeLists.txt

```diff
@@ -34,21 +34,3 @@ target_include_directories(
     PUBLIC
         ${LOGGING_INCLUDE_DIRS}
 )
```

Comment:
It may be value to set the absolute path for the `ROOT_CA_CERT_PATH` macro when building the mutual auth demo so that there isn't a relative path dependency which affects the directory from which the `mqtt_demo_mutual_auth` binary is run.

(The macro could be renamed to `AMAZON_ROOT_CA_PATH` instead to be more expressive, and also, so that the generic `ROOT_CA_CERT` macros in other demos are not affected. )

```
# Set the AMAZON_ROOT_CA_CERT_PATH configuration for the demo to the absolute path of the 	
# copied Amazon Root CA certificate.	
target_compile_definitions(	
    ${DEMO_NAME} 	
    PRIVATE 	
    AMAZON_ROOT_CA_CERT_PATH="$<TARGET_FILE_DIR:${DEMO_NAME}>/certificates/AmazonRootCA1.crt" )
```

Reply:
We discussed offline that we should have the path to the certificate passed as a CMake flag instead.